data/Record.js

```diff
@@ -43,20 +42,31 @@ export class Record {
      * requiring children to also be recreated.)
      *
      * @param {Object} c - Record configuration
+     * @param {Object} c.data - data for constructing the record.
      * @param {Object} c.raw - raw data for record.
      * @param {BaseStore} c.store - store containing this record.
```

Comment:
Realizing that LocalStore defines `idSpec`, which means that's what you need to be passing in here.  Yet `BaseStore` also declares interfaces that deal in Records/RecordSets.

We've discussed how we might want to unwind BaseStore, taking LocalStore -> Store and having it be the primary/parent store class.  I dunno if we want to deal with that now.

Alternatively we could move idSpec to BaseStore, or adjust the docs here to punt a bit and deal with LocalStore only.